### ISSUE TITLE
AttributeError: module 'gym.monitoring' has no attribute 'monitor'

### ISSUE THREAD

**Issue by jhurliman** [ppaquette/gym_pull#4]
```
import gym
import gym_pull
gym_pull.pull('github.com/ppaquette/gym-super-mario')
env = gym.make('ppaquette/SuperMarioBros-1-1-v0')
```

```
Traceback (most recent call last):
  File "hello.py", line 2, in <module>
    import gym_pull
  File ".../python3.6/site-packages/gym_pull/__init__.py", line 41, in <module>
    import gym_pull.monitoring.monitor
  File ".../python3.6/site-packages/gym_pull/monitoring/monitor.py", line 10, in <module>
    class Monitor(gym.monitoring.monitor.Monitor):
AttributeError: module 'gym.monitoring' has no attribute 'monitor'
```

**Comment by zergylord** [comment]
Same error here on python3.5. Is all of the gym stuff tested on python 3 or only python 2?

**Comment by zergylord** [comment]
After some digging around, I noticed gym has recently updated its gym.monitoring module such that gym.monitoring.monitor has been changed to gym.monitoring.monitor_manager. Tried just changing the import in the gym_pull source, but that led to more errors, likely stemming from monitor_manager having a different module structure than monitor did. Anyways, pretty sure that is the problem -- if someone with more knowledge of the gym_pull source code could take a look at it, I'm sure it'd be a quick fix.

**Comment by ghost** [comment]
Thanks for mentioning that recent change @zergylord

To resolve this issue, locate monitor.py in your python site-packages for gym-pull, and change the line:
`class Monitor(gym.monitoring.monitor.Monitor):`
to this:
`class Monitor(gym.monitoring.monitor_manager.MonitorManager):`

Once you've done this you may have to use:
`import ppaquette_gym_super_mario`

After all this, you'll see a ton of issues like this:
`[2017-01-15 01:23:17,589] DEPRECATION WARNING: env.spec.timestep_limit has been deprecated. Replace any calls to 'register(timestep_limit=200)' with 'register(tags={'wrapper_config.TimeLimit.max_episode_steps': 200)}', . This change was made 12/28/2016 and is included in gym version 0.7.0. If you are getting many of these warnings, you may need to update universe past version 0.21.1`

So we have to jump into ppaquette_gym_super_mario's __ init __.py and update the register() functions and make these changes:

```
#timestep_limit=9999999,
tags={'wrapper_config.TimeLimit.max_episode_steps': 9999999},

#timestep_limit=10000,
tags={'wrapper_config.TimeLimit.max_episode_steps': 10000},
```

I've confirmed these changes now allow me to call:
`env = gym.make('ppaquette/SuperMarioBros-1-1-v0')`

But I haven't done much else. Recent changes in gym may require more modifications in this repo. Just a heads up. Hopefully we can get this tested and submit a pull request to fix these issues for others. 

Edit: If anyone if having issues installing fceux on os x, check this out: http://anton.maurovic.com/posts/nintendo-nes-gamedev-part-1-setting-up/#installing-on-mac-os-x

**Comment by ppaquette** [comment]
Pushed v1.7 which fixes those issues. The gym package is changing too fast for me to keep up.

tags={'wrapper_config.TimeLimit.max_episode_steps': XXX} has been deprecated, and the new syntax is register(max_episode_steps=XXX)

The Monitor and MonitorManager object have been deprecated. Gym now uses a wrapper for the monitor.
